### PR TITLE
fix(plugins): Unloadable owner doesn’t WSOD displaying groupforumtopic

### PR DESCRIPTION
**Pull request by mrclay**

### CODE REVIEW

File: mod/groups/views/default/object/groupforumtopic.php [hunk 1]
```diff
@@ -9,10 +9,18 @@
 $topic = elgg_extract('entity', $vars, FALSE);
 
 if (!$topic) {
-	return true;
+	return;
 }
 
 $poster = $topic->getOwnerEntity();
+if (!$poster) {
+	elgg_log("The owner of entity {$topic->guid} cannot be laoded", 'WARNING');
```
Comment (juho-jaakkola) [inline mod/groups/views/default/object/groupforumtopic.php]: type: laoded

File: mod/groups/views/default/object/groupforumtopic.php [hunk 2]
```diff
@@ -9,10 +9,18 @@
...
+	if ($full) {
+		forward('', '404');
```
Comment (juho-jaakkola) [inline mod/groups/views/default/object/groupforumtopic.php]: I guess we might as well delete the orphaned topic

File: mod/groups/views/default/object/groupforumtopic.php [hunk 3]
```diff
@@ -9,10 +9,18 @@
 $topic = elgg_extract('entity', $vars, FALSE);
 
 if (!$topic) {
-	return true;
+	return;
 }
 
 $poster = $topic->getOwnerEntity();
+if (!$poster) {
+	elgg_log("User {$topic->owner_guid} could not be loaded, and is needed to display entity {$topic->guid}", 'WARNING');
```
Comment (mrclay) [inline mod/groups/views/default/object/groupforumtopic.php]: @juho-jaakkola added a bit more clarity here.

**Comment by ewinslow** [comment]
LGTM

**Pull request merged**